google-summer-of-code/GSOC-STUDENT-TEMPLATE.md

```diff
@@ -7,12 +7,16 @@
 - Start as early as possible; Make first contacts with the mentors before starting to fill in the proposal and make sure to discuss your project idea.
 - Share the first draft of your proposal as early as possible; Do not copy-paste from the official project description.
 - Share the proposal link as text, markdown, Google docs or anything else as long as it is easy to share, and will not require anyone to download something.
-- Final proposals should be submitted on the Google Summer of Code website before the deadline and after all the reviews have been done. 
+- Final proposals should be submitted on the Google Summer of Code website before the deadline and after all the reviews have been done.
 
-> Remove this section while you are writing the proposal. These pointers are purely meant for communicating the right details with the developers interested in proposing their ideas.
+### GSoC Template [Link](https://docs.google.com/document/d/1A6PnUWaxuFcl4Mwydgq7WluHJ1XkYVd5cX_-fSwgXpo/edit#)
```

Comment:
Please add this as a footnote for people to copy this if they are doing this on Google Docs

Reply:
ok, So should I put the link to the template in the footnotes?